tensorzero-core/tests/e2e/endpoints/stored_inferences/get_inferences.rs

```diff
@@ -1167,3 +1167,205 @@ pub async fn test_list_inferences_cursor_with_metric_ordering_fails() {
         "Request with cursor and metric ordering should fail"
     );
 }
+
+#[tokio::test(flavor = "multi_thread")]
+pub async fn test_get_by_ids_with_extra_body_and_inference_params() {
+    let http_client = Client::new();
+
+    // Create an inference with a nontrivial extra_body and inference params
+    let extra_body_value = json!([
+        {"pointer": "/test_field", "value": "test_value"},
+        {"pointer": "/nested/field", "value": {"key": "nested_value"}}
+    ]);
+
+    let inference_payload = json!({
```

Comment:
Consider building a structured Param struct then converting to json - with current setup, changes to the Param type will only cause runtime and not compile time errors.